app/frontend/src/components/ches/HistoryTable.vue

```diff
@@ -105,46 +105,19 @@ export default {
       return ['completed', 'cancelled'].includes(status);
     },
 
-    // get status details for each message in tansactions in store
-    populateTable() {
-      this.error = false;
-      this.loading = true;
-
-      try {
-        // TODO: Move this logic into vuex - do this BEFORE adding in remove/promote logic
-        // for each email transaction in store, get full status of messages
-        this.txIds.forEach(async (tx) => {
-          // if transaction isnt already in 'this.tableData' (vuex tableData property)
-          if (!this.tableData.find((o) => o.txId === tx.txId)) {
-            // get status details from CHES and add to tableData
-            const { data } = await chesService.getStatusByTransactionId(
-              tx.txId
-            );
-            this.addTableData(...data);
-          }
-        });
-      } catch (e) {
-        this.error = true;
-        this.showAlert({
-          type: 'error',
-          text: e,
-        });
-      }
-      this.loading = false;
-    },
-
     promoteMessage(msgId) {
       console.log('promoteMessage', msgId); // eslint-disable-line no-console
     },
 
     humanDateTime(timestamp){
       // date-fns docs: https://date-fns.org/v2.21.1/docs/Getting-Started
-      return timestamp === 0 ? '-' : format(new Date(timestamp), 'yyyy-MM-dd HH:mm');
+      return (timestamp === 0 || !timestamp) ? '-' : format(new Date(timestamp), 'yyyy-MM-dd HH:mm');
```

Comment:
Better to invert the ternary logic to make it more concise and leverage truthiness:
```js
return timestamp ? format(new Date(timestamp), 'yyyy-MM-dd HH:mm') : '-';
```